src/dbsync/api.rs

```diff
@@ -38,6 +38,27 @@ pub fn get_utxo_tokens(
     Ok(multi_assets)
 }
 
+pub fn get_utxo_tokens_for_txout(
```

Comment:
the function name is quite confusing, can we find something which describes the difference to the "get_utxo_tokens" which is a the function we want to use when we query token on UTxOs. 
This function is to query tokens from (S)TxOs.

Reply:
sure! get_stxo_tokens